imgparse/imgparse.py

```diff
@@ -93,25 +94,19 @@ def get_timestamp(image_path, exif_data=None, format_string="%Y:%m:%d %H:%M:%S")
         )
     except ValueError:
         raise ParsingError("Couldn't parse found timestamp with given format string")
-
+        
+    lat, lon = get_lat_lon(image_path, exif_data)
+    timezone = pytz.timezone(TimezoneFinder().timezone_at(lng=lon, lat=lat))
+    
     make, model = get_make_and_model(image_path, exif_data)
     if make == "Sentera":
         datetime_obj = pytz.utc.localize(datetime_obj)
     else:
-        lat, lon = get_lat_lon(image_path, exif_data)
-        try:
-            from timezonefinder import TimezoneFinder
-
-            timezone = pytz.timezone(TimezoneFinder().timezone_at(lng=lon, lat=lat))
-        except ImportError:
```

Comment:
removed this try/except block because I added timezonefinder to poetry since the sentera cameras will use it now as well

Reply:
See my comment about keeping `timezonefinder` optional. You'll need to then add this try/except back, but at the top level of the function so guards trying to parse the timestamp for all sensors

Reply:
https://github.com/SenteraLLC/py-image-metadata-parser/pull/63/commits/128f96f0bb5a2d923c00bdf2fdef61f780c3febb added the try/except back. Let me know if how I did it was correct

Reply:
Yep looks good